diff -urN ssidsniff-0.41.orig/cursesui.c ssidsniff-0.41/cursesui.c
--- ssidsniff-0.41.orig/cursesui.c	2005-04-09 07:45:03.000000000 +0200
+++ ssidsniff-0.41/cursesui.c	2005-04-25 11:46:33.000000000 +0200
@@ -8,7 +8,7 @@
 #endif
 
 #include <sys/param.h>
-#include <curses.h>
+#include <ncurses/ncurses.h>
 #include <stdlib.h>
 #include <string.h>
 #include <stdarg.h>
diff -burNp ssidsniff-0.53/configure.in ssidsniff-0.53-dud/configure.in
--- ssidsniff-0.53/configure.in	2007-06-28 05:18:47.000000000 +0200
+++ ssidsniff-0.53-dud/configure.in	2009-03-08 09:42:48.484662883 +0100
@@ -19,8 +19,8 @@ fi
 dnl curses compatible library checks
 AC_ARG_WITH(libcurses,
     [  --with-libcurses=DIR     Curses library location],
-    CFLAGS="-I${withval}/include $CFLAGS"
-    CPPFLAGS="-I${withval}/include $CPPFLAGS"
+    CFLAGS="-I${withval} $CFLAGS"
+    CPPFLAGS="-I${withval} $CPPFLAGS"
     LIBS="-L${withval}/lib -lcurses $LIBS"
 )
 AC_CHECK_HEADERS(curses.h,,
